src/app/models/admin_feedback.server.model.ts

```diff
@@ -0,0 +1,59 @@
+import { Mongoose, Schema } from 'mongoose'
+
+import { IAdminFeedbackModel, IAdminFeedbackSchema } from '../../types'
+
+import { USER_SCHEMA_ID } from './user.server.model'
+
+export const ADMIN_FEEDBACK_SCHEMA_ID = 'AdminFeedback'
+export const ADMIN_FEEDBACK_COLLECTION_NAME = 'adminfeedback'
```

Comment:
@mayying shall we start with recommending `snakecasing` collection names from this point onwards?

Reply:
wait but the mongo collections nomenclature is all lowercase (i googled before i think)

Reply:
As discussed, will be using `camelCase`! [b8c9de6](https://github.com/opengovsg/FormSG/pull/6454/commits/b8c9de66f9af9fcc6ac66858324f7d669727eed9)